stoix/systems/ppo/sebulba/ff_ppo.py

```diff
@@ -684,7 +706,7 @@ def run_experiment(_config: DictConfig) -> float:
         )
 
     # Get initial parameters
-    initial_params = unreplicate(learner_state.params)
+    initial_params = jax.device_put(learner_state.params, actor_devices[0])
```

Comment:
This is the only thing I am not sure of. Why do we put the initial params on the first actor device instead of the cpu or something.